terraform/region/secrets.tf

```diff
@@ -3,7 +3,7 @@
 # common
 resource "aws_secretsmanager_secret" "opg_lpa_common_admin_accounts" {
   name                           = "${local.account_name}/opg_lpa_common_admin_accounts"
-  tags                           = merge(local.default_opg_tags, local.admin_component_tag)
+  tags                           = local.admin_component_tag
```

Comment:
Just wondered what the reason was for reducing the number of tags here?

Reply:
The tags are duplicated at the provider level so Terraform "adds" these tags every time it runs. This just reduces noise in the Terraform output. I can remove this commit and add to a separate PR if necessary as it's only tangentially related to the main purpose of this PR

Reply:
Make sense :). I'd keep it in this PR, why not?  Just wondered the exact reason that's all :)